### DISCUSSION TITLE
Operating system error 67 Cannot open backup device '\\ServerName\R\TestDB.bak

### DISCUSSION THREAD

**Discussion by danielghub** [dokan-dev/dokany#1050] [Q&A] [answered]
Hi, im quite new on Dokan library nuget. We have this scenario where in the setup of our database is on AON, and once the secondary server tries to access this path '\\ServerName\R\TestDB.bak' its not being accessible. The **R** drive is created using mount function in the library and here is the snap of it

` this.Mount("R", DokanOptions.NetworkDrive, 5);  ` 

and this virtually created drive is being shared through managementclass library.

Its quite working well on different set of servers but we're experiencing this issue too frequent now especially on some newly provisioned servers. Was there any configuration should we check or any pre-requisites needed in order to have this virtually created drive accessible?

Thanks and please do let me know if you have questions regarding above.

**Comment by Liryna** [comment, marked as answer]
Have you tried to not use the network option and instead use the mount manager option ?
Or maybe the network drive with a uncname.

**Reply by danielghub** [reply]
Weird thing is, when we tried a different backup file particularly a smallest in size, we're able to access the path. Is there any some sort of TTL configured for the drive created by dokan? 

> Have you tried to not use the network option and instead use the mount manager option ?
Or maybe the network drive with a uncname.

@Liryna - Im sorry i couldn't catch up with your question above as this application lives for ages and dokan was integrated then and its been only awhile since i handled the application.

**Reply by Liryna** [reply]
Yes, there is a "TTL" for each request until Dolan decided the request took too much time to process the request and kills it. You can custom the value at mount time.

Please see the documentation of dokany, Dokan-dotnet: DokanOptions, DokanResetTimeout

**Reply by danielghub** [reply]
Thank you @Liryna , It seems that the full backup from the DB server took so much time and it cause the dokan to terminate its session. We're still checking what's wrong on the server side. Thank you for your response
